lockbox-ios/Presenter/ItemDetailPresenter.swift

```diff
@@ -151,9 +158,30 @@ class ItemDetailPresenter {
 
     private func setupNavigation() {
         self.view?.leftBarButtonTapped
-                .withLatestFrom(self.itemDetailStore.isEditing) { _, editing -> Action in
-                    return editing ? ItemDetailDisplayAction.viewMode : MainRouteAction.list
+                .withLatestFrom(self.itemDetailStore.isEditing) { _, editing -> Action? in
+                    if editing {
+                        self.view?.displayAlertController(
```

Comment:
I think we want to have more logic around whether there are changes to the fields in the editor view rather than blindly throwing up the dialog when leaving the editor view. @changecourse can you confirm?

Reply:
yeah that's definitely a unclear point when going off invision screens. If we need a "did anything change" check here that's fine but not sure what UX was intended.

Reply:
I think you are both right.

But for sake of getting to complete, I'm going to move this open question to confirm/do as a separate issue: #1071